src/gui/folderwatcher_win.cpp

```diff
@@ -141,6 +141,8 @@ void WatcherThread::watchChanges(size_t fileNotifyBufferSize,
 
             if (!skip) {
                 emit changed(longfile);
+            } else {
+                qCDebug(lcFolderWatcher) << "Skipping syncing of" << longfile;
```

Comment:
@camilasan you may want to add space after 'of'
other than that it's fine

Reply:
Actually not needed, qt logging facilities insert the spaces between arguments for you.